src/components/MapFieldValues/MapFieldValues.tsx

```diff
@@ -30,45 +45,45 @@ export const MapFieldValues = ({
         </PropertyRow>
       )}
       {usableFields.map((field) => {
-        const usedField = issue[field.key];
+        const fieldValue = issue[field.key];
 
         let content;
 
-        if (usedField === null || usedField === undefined) {
+        if (isNil(fieldValue)) {
           return <Property label={field.name} text={"-"} />;
         }
 
         switch (field.schema?.type) {
           case "progress":
-            content = <H2>{usedField.progress}</H2>;
+            content = <H2>{(fieldValue as Progress).progress}</H2>;
```

Comment:
We shouldn't need to type to Progress assuming `field.schema` is strongly typed and includes these "sub" types.

Reply:
I'm afraid I haven't found another way. Especially since there are no subtypes for custom fields in the documentation or the OpenAPI schema.